sdk/src/test/java/io/opentelemetry/sdk/trace/SpanBuilderSdkTest.java

```diff
@@ -420,40 +420,12 @@ public void parent_invalidContext() {
             tracerSdk.spanBuilder(SPAN_NAME).setParent(parent.getContext()).startSpan();
     try {
       assertThat(span.getContext().getTraceId()).isNotEqualTo(parent.getContext().getTraceId());
-      assertFalse(span.getParentSpanId().isValid());
+      assertFalse(span.toSpanData().getParentSpanId().isValid());
     } finally {
       span.end();
     }
   }
 
-  @Test
-  public void parent_timestampConverter() {
```

Comment:
Why do we remove these tests?

Reply:
as far as I could tell, these tests weren't actually providing any value, or were testing internals where we could be testing via the SpanData instead.

Reply:
If you think that noting that the clock isn't somehow swapped out by something is important, then I can re-instate them.

Reply:
I think the test were to prove that the Builder when creates a child Span inherits the Clock from the parent which I don't think it is tested with SpanData.

Reply:
Yeah, I guessed that was the purpose, but it seems like any other answer would be very strange. I can put them back if you feel they provide value.

Reply:
added back, with cleaned up naming.